command/ls/ls.go

```diff
@@ -74,7 +75,7 @@ func NewCommand(home string, docker client.CommonAPIClient, output terminal.Outp
 				}
 
 				if cmd.Flag("services").Value.String() == "true" {
-					if c.Labels[containerlabels.Type] != "dynamodb" && c.Labels[containerlabels.Type] != "mailhog" && c.Labels[containerlabels.Type] != "redis"  && c.Labels[containerlabels.Type] != "blackfire" {
+					if c.Labels[containerlabels.Type] != "dynamodb" && c.Labels[containerlabels.Type] != "mailhog" && c.Labels[containerlabels.Type] != "redis" && c.Labels[containerlabels.Type] != "blackfire" {
```

Comment:
Maybe add these services to an array and check if `c.Labels[containerlabels.Type]` exists in the array

Reply:
(yes I realize only a space changed here)

Reply:
Its rather ugly to use arrays in Go in that way, the best approach is a map and that's still not ideal... https://stackoverflow.com/a/10486196

Reply:
Happy to cleanup if you want though!

Reply:
My Golang-fu is weak, but feels like we're duplicating that logic in several places, so if there is an accepted way to clean it up, it'd be worth it.

Reply:
Resolved in https://github.com/craftcms/nitro/pull/433/commits/8b27be1e38e017536bbeb2dbdea81a794a9efdd2.